### PR TITLE
Remove Maven warnings

### PR DESCRIPTION
**Pull request by radtriste**
Those warnings are also causing problems with the Quarkus 3 OpenRewrite recipe ...

### CODE REVIEW

File: bom/drools-bom/pom.xml [hunk 1]
```diff
@@ -993,25 +993,6 @@
...
-        <artifactId>kie-drl-map-input-runtime</artifactId>
-        <version>${project.version}</version>
-        <scope>test</scope>
-      </dependency>
-      <dependency>
-        <groupId>org.kie</groupId>
-        <artifactId>kie-drl-kiesession-local-runtime</artifactId>
-        <version>${project.version}</version>
-        <scope>test</scope>
-      </dependency>
```
Comment (radtriste) [inline bom/drools-bom/pom.xml]: This part is also done by @evacchi in https://github.com/kiegroup/drools/pull/4838
I suggest we merge my PR after https://github.com/kiegroup/drools/pull/4838

**Comment by baldimir** [comment]
Jenkins run cdb

**Comment by baldimir** [comment]
As these are changes in the boms I think we should consider a full downstream build. What do you think? I started a compile downstream build at least, meanwhile.

**Comment by radtriste** [comment]
java heap space on examples ... happened on serverless workflow examples ...
this is more or less why we created 3 different jobs on examples (+ the time it takes to run) ...

**Review by baldimir — APPROVED** [review]

**Review by mariofusco — APPROVED** [review]

**Comment by radtriste** [comment]
jenkins rerun kogito-springboot-examples tests

**Pull request merged**